### PR TITLE
update test expectations

### PR DESCRIPTION
**Pull request by pq**
Should fix the build...

/cc @srawlins @bwilkerson

### CODE REVIEW

**Comment by srawlins** [comment]
Did you mean to CC Brandon Wang?

File: test/rules/prefer_contains_test.dart [hunk 1]
```diff
@@ -27,6 +27,7 @@ condition() {
 ''', [
       // No lint
       error(HintCode.UNUSED_LOCAL_VARIABLE, 41, 4),
+      error(CompileTimeErrorCode.ARGUMENT_TYPE_NOT_ASSIGNABLE, 77, 3),
```
Comment (srawlins) [inline test/rules/prefer_contains_test.dart:30]: I don't understand where this is coming from. I think the code is valid... `var next;` makes a dynamic, so the assignment should be valid.

Comment (pq) [reply test/rules/prefer_contains_test.dart:30]: The issue is that the list is of type `int` and we're passing in a String to `indexOf`.

image

**Comment by pq** [comment]
> Did you mean to CC Brandon Wang?

Whoops.  Nope!  Apologies for the noise, Brandon!

File: test/rules/prefer_contains_test.dart (hunk 1 above)
Comment (srawlins) [reply test/rules/prefer_contains_test.dart:30]: Ah I see. So strange that this wasn't picked up before...

**Review by srawlins — APPROVED** [review]

File: test/rules/prefer_contains_test.dart (hunk 1 above)
Comment (pq) [reply test/rules/prefer_contains_test.dart:30]: Yeah.  That's why I bumped the analyzer lower bound.  I'm guessing this got fixed in `4.3.1`.  (@scheglov?)

Comment (scheglov) [reply test/rules/prefer_contains_test.dart:30]: I don't remember fixing anything related recently.

**Pull request merged**